src/user/Admin/Tickets/TicketContent/TicketContent.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import "./TicketContent.scss";
+import Moment from "react-moment";
+import BadgeElement from "./BadgeElement";
+import { withRouter } from "react-router-dom";
+import { Image, Badge } from "react-bootstrap";
+import DataTable from "react-data-table-component";
+import customStyles from "./DataTableCustomStyles";
+import userIcon2 from "../../../../assets/images/userIcon2.jpg";
+import ChatBubbleOutlinedIcon from "@material-ui/icons/ChatBubbleOutlined";
+
+class TicketContent extends Component {
+  handleRowClick = (arg1) => {
+    this.props.viewTicket(arg1._id);
+  };
+
+  render() {
+    const CustomTitle = ({ row }) => {
+      return (
+        <div className="Ticket-dashboard-ticket">
+          <div className="status">
+            <BadgeElement ticketState={row.status} />
+          </div>
+          <div>
+            <div className="Ticket-dashboard-title">
+              {"# "}
+              {row.number}
+              {"  "}
+              {row.title}
+            </div>
+            <div className="Ticket-dashboard-shortDesciption">{`${row.shortDescription.slice(
```

Comment:
This was done because we did not want any upper bound on the number of characters in the short description but an upper bound of 100 characters was imposed on the characters to be displayed. This has been changed now, placed a check to ensure that short description is not more than 100 characters and removed the slice from here.